### ISSUE TITLE
TS1479, helmet typings issue

### ISSUE THREAD

**Issue by Uzlopak** [fastify/fastify-helmet#210]
### Prerequisites

- [X] I have written a descriptive issue title
- [X] I have searched existing issues to ensure the bug has not already been reported

### Fastify version

4.10.2

### Plugin version

10.1.0

### Node.js version

16.17.0

### Operating system

Linux

### Operating system version (i.e. 20.04, 11.3, 10)

22.04

### Description

It seems that we get typescript errors, because the typings from helmet are ecmascript and not commonjs

```typescript
uzlopak@Battlestation:~/workspace/nodenext-mvce$ npm run build

> nodenext-test@1.0.0 build
> tsc -p .

node_modules/@fastify/helmet/types/index.d.ts:2:62 - error TS1479: The current file is a CommonJS module whose imports will produce 'require' calls; however, the referenced file is an ECMAScript module and cannot be imported with 'require'. Consider writing a dynamic 'import("helmet")' call instead.

2 import helmet, { contentSecurityPolicy, HelmetOptions } from 'helmet';
                                                               ~~~~~~~~


Found 1 error in node_modules/@fastify/helmet/types/index.d.ts:2
```

### Steps to Reproduce

clone https://github.com/Uzlopak/nodenext-mvce/tree/helmet-issue
npm run build

### Expected Behavior

Should not throw

**Comment by kibertoad** [comment]
Would our fancy nodenext triplet help?

**Comment by kibertoad** [comment]
OK, I see https://github.com/fastify/fastify-helmet/pull/207/files
is it actually the cause for the problem?
@climba03003 thoughts?

**Comment by climba03003** [comment]
@kibertoad https://github.com/helmetjs/helmet/issues/389#issuecomment-1327745716

`TypeScript` is blaming the mis-match of types harder and harder. It is clearly a upstream issue.

**Comment by Uzlopak** [comment]
Opened a PR to fix this issue upstream https://github.com/helmetjs/helmet/pull/391

**Comment by ri5t3ai** [comment]
confirming it still persists 

```sh

node_modules/@fastify/helmet/types/index.d.ts:2:62 - error TS1479: The current file is a CommonJS module whose imports will produce 'require' calls; however, the referenced file is an ECMAScript module and cannot be imported with 'require'. Consider writing a dynamic 'import("helmet")' call instead.

2 import helmet, { contentSecurityPolicy, HelmetOptions } from 'helmet';

```

**Comment by climba03003** [comment]
I am going to lock this issue.
We know the issue is going but it cannot be fixed until `helmet` itself does the changes.

**Comment by Uzlopak** [comment]
@EvanHahn fixed this issue in v6.1.0 of helmet.

**Comment by kibertoad** [comment]
@Uzlopak should we bump minimum version?
